src/subdataframe/subdataframe.jl

```diff
@@ -71,7 +71,7 @@ function SubDataFrame(parent::DataFrame, row::Integer)
 end
 
 function SubDataFrame{S <: Integer}(parent::DataFrame, rows::AbstractVector{S})
-    return view(parent, Int(rows))
+    return SubDataFrame{Vector{Int}}(parent, convert(Vector{Int}, rows))
```

Comment:
We should only convert to `AbstractVector{Int}` since that's what the type accepts. I suspect the `Int(rows)` expression comes from an automated replacement which was done after `int(x)` was deprecated. This was effectively equivalent to `convert(AbstractVector{Int}, rows)`.

Also it seems it would more sense to remove this method and put this line into the generic constructor with `rows::T` above.